errors/index.js

```diff
@@ -0,0 +1,19 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
+
+exports.handlePsqlErrors = (err, req, res, next) => {
```

Comment:
We should avoid writing code we don't yet need and that has no application in the code we currently wrote - a 400 bad request isn't really something that would be applicable in this first GET request.

Re: 404: It's fine to rely on the edfault express 404 error handling, but you can also include a custom 404 handler in your app - then you should be testing for it, however.